ecommerce/migrations/0005_product_productimage.py

```
@@ -0,0 +1,54 @@
+        ),
+        migrations.CreateModel(
+            name='ProductImage',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('image', models.ImageField(help_text='Product image file', upload_to='ecommerce/products/images/')),
+                ('caption', models.CharField(blank=True, default='', help_text='Optional caption for the image', max_length=200)),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='When the image was uploaded')),
+                ('product', models.ForeignKey(help_text='Product this image belongs to', on_delete=django.db.models.deletion.CASCADE, related_name='images', to='ecommerce.product')),
+            ],
+            options={
+                'verbose_name': 'Product Image',
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)